### DISCUSSION TITLE
Can I use React 16 with Antd v5?

### DISCUSSION THREAD

**Discussion by sdt-jchau** [ant-design/ant-design#42504] [Q&A]
Looking at Antd v5 package.json, it lists React 18 as a dev dependency but React 16 as a peer dependency.  When I upgrade my React 16 project from v4 to v5, I get a lot of compiler errors with RC and Antd .d.ts files referencing React 18 API.

**Comment by RedJue** [comment]
Antd v5 supports a minimum of react 16.8, because this version begins to enable the hooks API.

**Reply by sdt-jchau** [reply]
but when using Typescript, the type files are referencing React 18 APIs so that means my project also needs to be on React 18 right?  If not, I get tons of compiler errors.

**Comment by sdt-jchau** [comment]
Resolved by adding `"skipLibCheck": true,` to tsConfig.json and upgrade Typecript to 4.x
